text/0000-async_await.md

```diff
@@ -206,29 +234,12 @@ fn foo<'a>(arg1: &'a str, arg2: &str) -> impl Future<Item = usize> + 'a {
     // do some initialization using arg2
 
     // closure which is evaluated immediately
-    (async move || {
-         // asynchronous portion of the function
-    })()
-}
-```
-
-The futures crate will provide an `async_block!` macro to make this cleaner:
-
-```rust
-// only arg1's lifetime is captured in the returned future
-fn foo<'a>(arg1: &'a str, arg2: &str) -> impl Future<Item = usize> + 'a {
-    // do some initialization using arg2
-
-    // closure which is evaluated immediately
-    async_block! {
+    async move {
```

Comment:
Should `move` go before `async` for consistency with closures?

Reply:
I imagined that closures would also be written `async move |x| { ... }`.

Reply:
Ah yeah, that's a different kind of consistency that may be more important? Overall as long as we have a consistent ordering everywhere I'm happy. (Just dealt with the lack of an enforced ordering in C++...)

Reply:
Interesting. We don't have const or unsafe closures so `async` would be first. I think `async move`, `const move`, `unsafe move` works nicely. 👍

Reply:
`const unsafe async move |x| { ... }`